sdk/messaging/azservicebus/migrationguide.md

```diff
@@ -64,29 +64,33 @@ Sending messages in batches is similar, except that the focus has been moved mor
 towards giving the user full control using the `MessageBatch` type.
 
 ```go
-batch, err := sender.NewMessageBatch(context.TODO(), nil)
+// Create a message batch. It will automatically be sized for the Service Bus
+// Namespace's maximum message size.
+messageBatch, err := sender.NewMessageBatch(context.TODO(), nil)
 
-// can be called multiple times
-err := batch.AddMessage(&azservicebus.Message{
-  Body: []byte("hello world"),
+if err != nil {
+  panic(err)
+}
+
+// Add a message to our message batch. This can be called multiple times.
+err = messageBatch.AddMessage(&azservicebus.Message{
+    Body: []byte(fmt.Sprintf("hello world")),
 })
 
-if err != nil {
-  switch err {
-  case azservicebus.ErrMessageTooLarge:
-    // At this point you can do a few things:
-    // 1. Ignore this message
-    // 2. Send this batch (it's full) and create a new batch.
-    //
-    // The batch can still be used after this error if you have
-    // smaller messages you'd still like to add in.
-    fmt.Printf("Failed to add message to batch\n")
-  default:
-    exitOnError("Error while trying to add message to batch", err)
+if err == azservicebus.ErrMessageTooLarge {
```

Comment:
if `errors.Is(err, azservicebus.ErrMessageTooLarge)`

Reply:
Fixed (also had the same construct in readme.md)